src/gui/shareusergroupwidget.cpp

```diff
@@ -36,6 +38,8 @@
 #include <QFileInfo>
 #include <QAbstractProxyModel>
 #include <QCompleter>
+#include <qboxlayout.h>
```

Comment:
@FlexW Why not use `#include <QBoxLayout` as it is described in the manual? https://doc.qt.io/qt-5/qboxlayout.html same for `qicon.h`.

Reply:
That was the automatic header inseration. No idea, why they have this indirection inside Qt. Makes always more work for me:)

Reply:
@FlexW Most-likely it is done to make sure include is still valid if they ever change the name of the header. They recommend using `#include <QWhatEver>` in their coding style https://doc.qt.io/qtcreator-extending/coding-style.html#including-headers